src/AFUP/HaphpyBirthdayBundle/Resources/translations/messages.fr.yml

```diff
@@ -116,17 +116,17 @@ about:
         limit: >
             <p>Les Contributions peuvent être <i>uploadées</i> jusqu'à la date limite du %limit_date%.</p>
         respect: >
-            <p>Les médias envoyés par les contributeurs au projet doivent exprimer de la gratitude
-            envers le langage PHP ainsi que sa communauté</p>
+            <p>Les médias envoyés par les contributeurs au projet sont invités à exprimer de la gratitude
+            envers le langage PHP ainsi qu'à sa communauté</p>
             <p>Les Contributions doivent naturellement respecter tout le monde.</p>
         ad_free: >
-            <p>La vidéo commemorative final ne contiendra pas de publicité. Les sociétés
+            <p>La vidéo commémorative finale ne contiendra pas de publicité. Les sociétés
             ne sont pas autorisées à utiliser le présent projet comme support publicitaire.</p>
-            <p>Une société peut utiliser tout compte de réseau social pour télécharger une
-            Contribution mais le média envoyé ne peut contenir juste une publicité pour celle-ci.
+            <p>Une société peut utiliser, via le réseau social de son choix, une
+            contribution ; le média envoyé ne devra pas explicitement faire de la pubilicité en faveur de la société.
```

Comment:
Thanks @Hellslicer I merged the PR already (probably when you were commenting). I'll correct this ASAP.